kore/app/repl/Main.hs

```diff
@@ -185,85 +187,84 @@ mainWithOptions
         , scriptModeOutput
         , outputFile
         , koreLogOptions
-        }
-  = do
-    exitCode <-
-        withBugReport Main.exeName BugReportOnError $ \tempDirectory ->
-            withLogger tempDirectory koreLogOptions $ \actualLogAction -> do
-            mvarLogAction <- newMVar actualLogAction
-            let swapLogAction = swappableLogger mvarLogAction
-            flip runLoggerT swapLogAction $ do
-                definition <- loadDefinitions [definitionFileName, specFile]
-                indexedModule <- loadModule mainModuleName definition
-                specDefIndexedModule <- loadModule specModule definition
+        } =
+        do
+            exitCode <-
+                withBugReport Main.exeName BugReportOnError $ \tempDirectory ->
+                    withLogger tempDirectory koreLogOptions $ \actualLogAction -> do
+                        mvarLogAction <- newMVar actualLogAction
+                        let swapLogAction = swappableLogger mvarLogAction
+                        flip runLoggerT swapLogAction $ do
+                            definition <- loadDefinitions [definitionFileName, specFile]
+                            indexedModule <- loadModule mainModuleName definition
+                            specDefIndexedModule <- loadModule specModule definition
```

Comment:
This is a particularly bad example of indentation creep, but at least this would encourage us to factor out `withBugReport` and `withLogger`.